Dockerfile.rhtap

```diff
@@ -1,9 +1,9 @@
-FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_1.22 as builder
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_1.23 as builderi
```

Comment:
Did you mean to rename builder to builder1?  The name builder is still used later

Reply:
good catch! thanks!

Fixed